src/config/mod.rs

```diff
@@ -50,6 +50,14 @@ pub fn internal_client_connect_timeout() -> u64 {
     u64_with_default("INTERNAL_CLIENT_CONNECT_TIMEOUT", 1000)
 }
 
+pub fn request_error_cache_timeout() -> usize {
+    usize_with_default("REQS_ERROR_CACHE_DURATION", 60)
+}
+
+pub fn info_error_cache_timeout() -> usize {
+    usize_with_default("INFO_ERROR_CACHE_DURATION", 60 * 60 * 24)
```

Comment:
Should we use the respective non-error timeout as a default? ... hmm maybe not ... not sure :D

Reply:
We can tune this later I suppose :) I will document this timeouts in #278